crazyflie_cpp/include/crazyflie_cpp/Crazyflie.h

```diff
@@ -242,6 +264,7 @@ class Crazyflie
   template<typename T>
   friend class LogBlock;
   friend class LogBlockGeneric;
+  friend class CrazyflieROS;
```

Comment:
crazyflie_cpp should work without ROS, i.e. there should be no "forward" dependency to ROS within crazyflie_cpp. One way would be to make it public, or to use a callback as suggested in the issue.